documentation/Using-Azure-TestFramework.md

```diff
@@ -189,3 +193,11 @@ Setting `Environment` in the connection string will result in the following defa
         GraphTokenAudienceUri = "https://graph.ppe.windows.net/"
         DataLakeStoreServiceUri = "https://caboaccountdogfood.net"
         DataLakeAnalyticsJoAbndCatalogServiceUri = "https://konaaccountdogfood.net"
+
+##### Environment = Custom
+When specified, test framework expect all Uri's to be provided by the user as part of the connection string.
+
+What is also supported is as below (connections string example)
+>SubscriptionId=subId;Environment=Prod;AADAuthUri=customAuthUri;ResourceManagementUri=CustomR>esourceMgmtUri
```

Comment:
Perhaps:  "It is also supported to override one or more settings for a selected environment, as in the following example"